overlay/usr/lib/bitkey.d/inithooks-preseed

```diff
@@ -7,5 +7,6 @@ export ROOT_PASS=$MASTERPASS
 export USER_PASS=$MASTERPASS
 export SEC_UPDATES=SKIP
 export HUB_APIKEY=SKIP
+export SEC_ALERTS=SKIP
```

Comment:
Considering the use case for BitKey I'm not convinced that skipping security updates on firstboot is a good idea...